l10n/en/firefox/features/password-manager-2023.ftl

```diff
@@ -19,7 +19,8 @@ password-manager-with-a-free-mozilla-account = With a <a href="{ $fxa }">free {
 # Obsolete string
 # Variables:
 #   $fxa (url) = link to https://www.mozilla.org/firefox/accounts/
-password-manager-with-a-free-firefox-account = With a <a href="{ $fxa }">free { -brand-name-firefox } account</a> you can securely sync your passwords across all your devices.
+password-manager-with-a-free-firefox-account = With a <a href="{ $fxa }">free { -brand-name-firefox-account }</a> you can securely sync your passwords across all your devices.
```

Comment:
This needs a new ID, even if it's obsolete (there are translations with a different term, and it will create warnings all over the place).

But, if it's indeed obsolete, we should leave it as is.

Reply:
This change was just reverting the string to how it was before the initial Moz accounts update merge. Comment with some more info mentioned here: https://github.com/mozilla/bedrock/pull/13706#discussion_r1333049850

Reply:
The string that we [currently expose](https://github.com/mozilla-l10n/www-l10n/blob/master/en/firefox/features/password-manager-2023.ftl#L22C79-L22C110) has `term + account`, and [all translations](https://transvision.flod.org/string/?entity=mozilla_org/en/firefox/features/password-manager-2023.ftl:password-manager-with-a-free-firefox-account&repo=mozilla_org) have the same, with the exceptions of locales managed outside of Pontoon (ar, de, fr so far).

We should leave it as is.